packages/okta-react/test/e2e/harness/e2e/page-objects/app.po.js

```diff
@@ -20,15 +20,15 @@ export class AppPage {
   waitUntilVisible() {
     const loginExists = ExpectedConditions.presenceOf(this.getLoginButton());
     const logoutExists = ExpectedConditions.presenceOf(this.getLogoutButton());
-    browser.wait(ExpectedConditions.or(loginExists, logoutExists), 5000);
+    browser.wait(ExpectedConditions.or(loginExists, logoutExists), 15000);
```

Comment:
Would it make sense to pull all these 15000 into a single constant that can be moved up/down as we deem reasonable?

Reply:
yes. it would. i was being lazy (like the last guy) ;)